libs/pages/elewa/invest/src/lib/components/elewa-invest-hero/elewa-invest-hero.component.html

```diff
@@ -0,0 +1,12 @@
+<div id="elewa-hero">
+  <img class="backgroundImage" [src]="backgroundImg" alt="top banner hero image" />
+
+  <div class="container">
+    <div>
+      <div class="content">
+        <h2 class="subtitle">{{ subtitle }}</h2>
+        <h1 class="title">{{ title }}</h1>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
This is not how to reuse a component.
Use the component tag and pass inputs as parameters on the opening tag

<elewa-group-elewa-hero  [titleText]="Your Title"  ></elewa-group-elewa-hero>